### ISSUE TITLE
AFECOI appears to use wrong hydraulic diameter

### ISSUE THREAD

**Issue by jasondegraw** [NREL/EnergyPlus#7012]
Issue overview
--------------
It appears to me that the AFECOI routine in AirflowNetwork is using data from the wrong component array. From around line 2518 in AirflowNetworkSolver.cc:

```
        // Get component properties
        CompNum = AirflowNetworkCompData(j).TypeNum;
        ed = Rough / DisSysCompDuctData(CompNum).D;
        area = pow_2(DisSysCompCoilData(CompNum).D) * Pi;
        ld = DisSysCompCoilData(CompNum).L / DisSysCompCoilData(CompNum).D;
        g = 1.14 - 0.868589 * std::log(ed);
        AA1 = g;
```

The relative roughness calculation (`ed`) on the third line above is using the diameter from a duct object: `DisSysCompDuctData(CompNum).D`.

The fix is simple (replace `Duct` with `Coil`), but it does cause diffs.

@lgu1234 Do you agree that this is a bug?

### Details
Some additional details for this issue (if relevant):
 - Platform: All
 - Version of EnergyPlus: 9.0.1

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by lgu1234** [comment]
@jasondegraw I agree with you. A good catch.

**Comment by Myoldmopar** [comment]
Closed via #7016
